atlp-rwanda/atlp-pulse-bn
# #426 implement coordinator notification

### PR Description
This PR adds the functionality to notify coordinators of important events and updates within the platform.

### Description of tasks  and Functionality
- Added graphql type, query, mutation, and subscription to support notification system.
- Utilizes a global function pushNotification to create a notification and notify coordinator
- Delivers real-time notifications to coordinators using GraphQL subscriptions.
- Ensure that coordinator receive immediate updates on action on platform


### How should this be manually tested?
- Clone this repository and navigate to the project directory
- Ensure that you have set your .env file using .env.example.
```bash
  git clone https://github.com/atlp-rwanda/atlp-pulse-bn.git
  cd atlp-pulse-bn
  git checkout ft-coordinator-notification
  npm i
  npm run dev
  ```
